google/auth/transport/aiohttp_requests.py

```diff
@@ -230,8 +280,16 @@ async def request(
                 transmitted. The timout error will be raised after such
                 request completes.
         """
+        # Headers come in as bytes which isn't expected behavior (resumable)
```

Comment:
What do you mean by "isn't expected behavior"? Can you clarify to explain why the lines below are necessary?

Reply:
This isn't expected as most headers are of a str type. It is odd that some come through as bytes. This may be a bandaid, or may be entirely necessary :(

Reply:
Added more explanation to why we are doing this. Inconsistencies in the header value types for some cases in the resumable library, which is why we have this check.